uwblueprint/planet-read
# Merge SignUp and Login to be one component

## Notion ticket link



[Text in login fields should stay when switching to sign up](https://www.notion.so/uwblueprintexecs/Text-in-login-fields-should-stay-when-switching-to-sign-up-f3d49f715be44ff8aeb7182cab564b0e)



## Implementation description

- Merged signup page into login page
- Deleted signup component and route
- Added isSignup state to distinguish between login and signup



## Steps to test

1. Go to http://localhost:3000/signup, should get 404 not found
2. Go to http://localhost:3000/login, should get login page
3. Enter email and password
4. Click signup, should get signup page with email and password
5. Change email and password, enter names
6. Click login, should get login page with changed email and password
7. Click signup, should get signup page with email, password, and names
8. Click login and change email and password to working ones
9. Click login, should be logged in
10. Reset by making a new window and go to http://localhost:3000/login
11. Click signup and fill in fields
12. Signup should be attempted (CORS/network/graphql/internal server errors will be logged to console)



## What should reviewers focus on?

- Are the pages merged properly
- Are functions working as expected

## Checklist
